modules/issue_tracker/php/issuewatchermapper.class.inc

```diff
@@ -27,7 +26,7 @@ use \LORIS\Data\{Mapper, DataInstance
  * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link       https://www.github.com/aces/Loris/
  */
-class IssueWatcherAnonymizer implements Mapper
+class IssueWatcherMapper implements Mapper
```

Comment:
The comments below are still wrong and describe this as an anonymizer/anonymized data.